configs/sagan/README.md

```diff
@@ -0,0 +1,59 @@
+# Self-attention generative adversarial networks
+
+## Introduction
+<!-- [ALGORITHM] -->
+```latex
+@inproceedings{zhang2019self,
+  title={Self-attention generative adversarial networks},
+  author={Zhang, Han and Goodfellow, Ian and Metaxas, Dimitris and Odena, Augustus},
+  booktitle={International conference on machine learning},
+  pages={7354--7363},
+  year={2019},
+  organization={PMLR}
+}
+```
+<div align="center">
+  <b> Results from our SAGAN trained in CIFAR10</b>
+  <br/>
+  <img src="https://user-images.githubusercontent.com/28132635/127619657-67f2e62d-52e4-43d2-931f-6d0e6e019813.png" width="400"/>
+</div>
+
+|  Models   | Dataset  | Inplace ReLU | n_disc | Total Batchsize (BZ_PER_GPU \* NGPU \* N_GRAD_ACCU) | Total Iters* |                                                                               Best IS (Iter)                                                                                |                                                                               Best FID (Iter)                                                                                |                                                                       Config                                                                        |                                                                               Log                                                                               |
+| :-------: | :------: | :----------: | :----: | :-------------------------------------------------: | :----------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| sagan_32  | CIFAR10  |     w/o      |   5    |                       64x1x1                        |    500000    |        [ 9.3217 (400000) ](https://download.openmmlab.com/mmgen/sagan/sagan_cifar10_32_lr2e-4_ndisc5_b64x1_woReUinplace_is-iter400000_20210730_125743-4008a9ca.pth)         |        [ 9.4252 (480000) ](https://download.openmmlab.com/mmgen/sagan/sagan_cifar10_32_lr2e-4_ndisc5_b64x1_woReUinplace_fid-iter480000_20210730_125449-d50568a4.pth )        |        [config](https://github.com/open-mmlab/mmgeneration/tree/master/configs/sagan/sagan_32_woReLUinplace_cifar10_b64x1_lr-2e-4_ndisc5.py)        |       [Log](https://download.openmmlab.com/mmgen/sagan/sagan_cifar10_32_lr2e-4_ndisc5_b64x1_woReUinplace_20210730_125449_fid-d50568a4_is-04008a9ca.json)        |
```

Comment:
If `N_GRAD_ACCU` is always 1, we can just ignore it in this benchmark.

Reply:
`N_GRAD_ACCU` is set for our ongoing setting, which `N_GRAD_ACCU=2`.